### PR TITLE
Add Whales

### PR DESCRIPTION
**Pull request by mayra-cabrera**
**Project: https://github.com/Gueils/whales**

**Description: Whales is a tool to automatically dockerize your applications. It works as command line interface that tames the [Belugas](#belugas-analyzers) features analyzers and outputs necessary Docker files to run your application with Docker**

### CODE REVIEW

**Comment by Moshe-Immerman** [comment]
Why is this awesome? It has only 3 contributors and a handful of stars?

**Comment by veggiemonk** [comment]
@mayra-cabrera I see you've put some work into that project and it shows. 
Now the examples are a bit inconsistent, sometimes the images is `whalesil/whales:latest` but in the GIF the image is `acalialabs/whales`. That confuses people.
Also it would be nice to explain some of those parameters and what they are meant for.
The "usage" section needs more work!

We value people's time a lot so we want them to find the best projects there is.
Our goals is also to help developers to make great projects.
So a good start would be to look at other projects and take good ideas from them.
Docker-sync has a very simple website hosted on github pages, that really improves the reach of the project (more people come and check it out). http://docker-sync.io/
Some developers go really the extra miles to explain what their project is all about such as https://github.com/grammarly/rocker

So let us know if you need any help. Then come back here and ping us. All right?
Once you do that, the project will be on the list for sure!

Have a good day and keep on making great software!

The awesome docker team

@vegasbrianc @gesellix @dmitrytokarev  @devhkr @Moshe-Immerman @veggiemonk


@Moshe-Immerman you should talk nicely to people, especially if you don't want to see your maintainer's right revoked! You are here to help people, not to judge them! 
Read the contribution guidelines before dismissing a project and this https://github.com/veggiemonk/awesome-docker/pull/289#issuecomment-285608004

File: README.md [hunk 1]
```diff
@@ -256,6 +256,7 @@ Services to securely store your Docker images.
 * [portainer](https://github.com/duedil-ltd/portainer) - Apache Mesos framework for building Docker images by [@duedil-ltd](https://github.com/duedil-ltd)
 * [rocker](https://github.com/grammarly/rocker) - Extended Dockerfile builder. Supports multiple FROMs, MOUNTS, templates, etc. by [grammarly](grammarly).
 * [SkinnyWhale](https://github.com/djosephsen/skinnywhale) Skinnywhale helps you make smaller (as in megabytes) Docker containers.
+* [Whales](https://github.com/Gueils/whales) - A tool to automatically dockerize your applications by [@icalialabs](https://github.com/IcaliaLabs).
```
Comment (veggiemonk) [inline README.md]: Well done, nice project! see the comment for improvements

**Comment by agebhar1** [comment]
Sounds interesting @mayra-cabrera! Beside @veggiemonk hints the link to 'Belugas Java' is broken.

**Comment by Moshe-Immerman** [comment]
@mayra-cabrera I hope you didn't take offense to my terse reply - it certainly wasn't intended that way.  May I also suggest that your post a blog or 2 about the project and how it came about so that you can get some more eyes on it?
@veggiemonk  thanks I have saved the comment as reply template for future

**Comment by veggiemonk** [comment]
@Moshe-Immerman No worries. I'm glad you are on board with us :D

**Comment by mayra-cabrera** [comment]
@veggiemonk @agebhar1 @Moshe-Immerman thanks for your suggestions! I'll get back to you once I'm done with the changes

**Comment by mayra-cabrera** [comment]
hey @veggiemonk @agebhar1 @Moshe-Immerman thanks for your patience.  

We follow your suggestions and improve the readme outline by adding the following sections: "_How does it work_", "_Architecture_" & "_Command explanation_": https://github.com/Gueils/whales. I hope is more consistent and gives you a better idea of what the project is all about.

Also to make it more friendly and reach more people, we create a website for it: http://www.whales.mx

What do you think? Is this ready to be merged? :)

**Comment by agebhar1** [comment]
Hey @mayra-cabrera thank you for your improvements! I can earliest have a look at Sunday. Maybe one of the other team members @veggiemonk @devhkr @vegasbrianc @gesellix @Moshe-Immerman can have a look.

**Comment by veggiemonk** [comment]
@mayra-cabrera good work.

**Pull request merged**